<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaletteSettingsForm</class>
 <widget class="QWidget" name="PaletteSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>448</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Настройки контура</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="ScrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="ScrollAreaContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>319</width>
        <height>430</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="SelectableCheckBox">
         <property name="text">
          <string>Выбирать ячейки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="LightEnabledCheckBox">
         <property name="text">
          <string>Включить освещение</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="TransparencyLayout">
         <item>
          <widget class="QLabel" name="TransparencyLabel">
           <property name="text">
            <string>Прозрачность:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="TransparencySlider">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>9</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
